### PR TITLE
Feature/improve pull to refresh animation

### LINKED ISSUE #82: To improve the 'pull to refresh' animation in the screens where this action is supported

### PR DESCRIPTION
**Pull request by ahmed-tarek-salem**
This pull request improves the pull-to-refresh animation to match native iOS appearance and behavior on iOS platforms, as discussed in #82.

- The pull-to-refresh appearance is now adaptive based on the platform.
- When an app bar is present on the screen, the refresh indicator animation starts below the app bar.

Fixes #82

### CODE REVIEW

File: lib/pages/main/tab_wallet_contents.dart [hunk 1]
```diff
@@ -333,8 +335,8 @@ class _TabWalletContentsState extends State<TabWalletContents> {
                                       vertical:
                                           ThemePaddings.normalPadding / 2),
                                   child: AccountListItem(
-                                      item: appStore.currentQubicIDs[index])));
-                        }, childCount: appStore.currentQubicIDs.length),
+                                      item: appStore.currentQubicIDs[0])));
```
Comment (sallymoc) [inline lib/pages/main/tab_wallet_contents.dart:338]: Is this intended? or was used for your testing?

**Comment by sallymoc** [comment]
Please change destination branch to develop. We are using Gitflow workflow.

File: lib/pages/main/tab_wallet_contents.dart (hunk 1 above)
Comment (ahmed-tarek-salem) [reply lib/pages/main/tab_wallet_contents.dart:338]: Thank you for catching that! it was used only for testing.
I'll remove it and push an update shortly.

**Comment by sallymoc** [comment]
image
can we have the loading in the position pointed by the arrow in the image above?

**Comment by sallymoc** [comment]
@icyblob I could do the squash here directly when merging

**Pull request merged**